src/pages/index.js

```diff
@@ -135,12 +136,14 @@ const useStyles = makeStyles((theme) => ({
   },
   notification: {
     marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(1),
   },
   notificationsContainer: {
-    width: 350,
+    width: 400,
     display: 'flex',
     justifyContent: 'flex-end',
     paddingRight: theme.spacing(1),
+    flexDirection: 'column',
```

Comment:
With the flex change, do you have to add `alignItems` to stick this to the right? Without testing, the problem I'm imagining is that a wider user menu will pull the container to the left.

Reply:
Ah, I didn't because the other notification took the full width so they were aligned. I can add a property here though.